tests/virtualization/universal/storage.pm

```diff
@@ -25,6 +25,7 @@ sub run_test {
     my ($self) = @_;
 
     record_info "Prepare";
+    start_guests();
```

Comment:
All guests should be already running at this point. Maybe this line should be elsewhere?

Reply:
because running guests are shutdown in previous hotplugging test module. The recent change to this module:
## 5. Cleanup
    shutdown_guests();
So I have to start the guests again in storage.

Reply:
Not everywhere. Can you please at least check if guests are running first?

Reply:
No problem.